docs-source/content/faq/external-clients.md

```diff
@@ -0,0 +1,202 @@
+
+If an EJB or JMS service is running on an Admin Server, then you can skip the rest of this section and use the `spec.adminServer.adminService.channels` domain resource attribute to have the operator create a Node Port for you. See [Reference - Domain resource]({{<relref "/reference/domain-resource/_index.md">}}). Otherwise, if the EJB or JMS service is running in a WebLogic cluster or stand-alone WebLogic managed server, and you desire to provide access to the service using a NodePort, then the NodePort must be deployed 'manually' - see the following sample and table.
+
+> __NOTE:__ Setting up a NodePort usually also requires setting up a custom network channel. See [Adding a WebLogic Custom Channel](#adding-a-weblogic-custom-channel) above.
+
+#### Sample NodePort Resource
+
+The following NodePort YAML deploys an external node port of `30999` and internal port `7999` for a domain UID of `DOMAIN_UID`, a domain name of `DOMAIN_NAME`, and a cluster name of `CLUSTER_NAME`. It assumes that `7999` corresponds to a `T3` protocol port of a channel that's configured on your WebLogic cluster.
+
+```
+apiVersion: v1
+kind: Service
+metadata:
+  namespace: default
+  name: DOMAIN_UID-cluster-CLUSTER_NAME-ext
+  labels:
+    weblogic.domainUID: DOMAIN_UID
+spec:
+  type: NodePort
+  externalTrafficPolicy: Cluster
+  sessionAffinity: ClientIP
+  selector:
+    weblogic.domainUID: DOMAIN_UID
+    weblogic.clusterName: CLUSTER_NAME
+  ports:
+  - name: myclustert3channel
+    nodePort: 30999
+    port: 7999
+    protocol: TCP
+    targetPort: 7999
+```
+
+#### Table of NodePort Attributes
+
+|Attribute|Description|
+|---------|-----------|
+|metadata.name|For this particular use case, the NodePort name can be arbitrary as long as it is DNS compatible. But, as a convention it's recommended to use `DOMAIN_UID-cluster-CLUSTER_NAME-ext`. To ensure the name is DNS compatible, use all lower case and convert any underscores (`_`) to dashes (`-`).|
+|metadata.namespace|Must match the namespace of your WebLogic cluster.|
+|metadata.labels|Optional. It's helpful to set a weblogic.domainUid so that cleanup scripts can locate all Kubernetes resources associated with a particular domain UID.|
+|spec.type|Must be `NodePort`|
+|spec.externalTrafficPolicy|Set to `Cluster` for most use cases. This may lower performance, but ensures that a client that attaches to a Node without any pods that match the `spec.selector` will be rerouted to a node with pods that do match. If set to `Local`, then connections to a particular Node will only route to that Node's pods and will fail if the Node doesn't host any pods with the given `spec.selector`. It's recommended for clients of a `spec.externalTrafficPolicy: Local` NodePort to use a URL that resolves to a list of all nodes such as `t3://mynode1,mynode2:30999` so that a client connect attempt will implicitly try mynode2 if mynode1 fails (alternatively, use a round-robin DNS address in place of `mynode1,mynode2`).|
```

Comment:
No reason to capitalize Node in this description text.
mynode1 -> `mynode1`
mynode2 -> `mynode2`